src/thermal_save_csv.py

```diff
@@ -203,7 +203,7 @@ def main():
 
         # instead of taking the ambient temperature over one frame of data take it over a set amount of time
         MAXTEMP = float(np.mean(mode_list)) + AMBIENT_OFFSET
-        pixels = [map_value(p, np.mean(mode_list) + 2, MAXTEMP, 0,
+        pixels = [map_value(p, np.mean(mode_list) + 1, MAXTEMP, 0,
```

Comment:
(ambient temp + 1, MAXTEMP) temperature scale is mapped to (0, COLORDEPTH - 1) grey scale